### ISSUE TITLE
Kaggle API key with Colab

### ISSUE THREAD

**Issue by AnthonyByansi** [googlecolab/colabtools#3250]
I'm running google colab, authenticated it with kaggle privated key (Json file)  with the name to the kaggle account. I tried using pip to install tensorflow and Keras and seen to get a simillar error .
![Capture](https://user-images.githubusercontent.com/101401469/204252698-c8a95a3b-6931-40c5-a808-6ad26743fb60.PNG)

**Comment by mco-gh** [comment]
Can you share a minimal reproducible notebook?

**Comment by AnthonyByansi** [comment]
`import numpy as np
import pandas as pd 
import random
import os
import matplotlib.pyplot as plt
import seaborn as sns
import keras.applications.xception as xception
import zipfile
import sys
import time
import tensorflow as tf
import tensorflow.keras as keras
import re

from PIL import Image
from keras.layers import Input, Conv2D, Dense, Flatten, MaxPooling2D, Input, GlobalAveragePooling2D
from keras.layers.experimental.preprocessing import Normalization
from keras.models import Model, Sequential
from keras.preprocessing import image
from keras.utils import to_categorical
from keras.layers import Lambda
from keras.callbacks import EarlyStopping
from sklearn.model_selection import train_test_split
from sklearn.metrics import classification_report

print('setup successful!')`

**Comment by mco-gh** [comment]
Try changing this:
```
from keras.layers.experimental.preprocessing import Normalization
```
to this:
```
from keras.layers import Normalization
```
